config/src/main/java/org/springframework/security/config/annotation/web/configurers/ExceptionHandlingConfigurer.java

```diff
@@ -127,6 +132,18 @@ public ExceptionHandlingConfigurer<H> defaultAccessDeniedHandlerFor(AccessDenied
 		return this;
 	}
 
+	public ExceptionHandlingConfigurer<H> defaultAuthenticationEntryPointFor(AuthenticationEntryPoint entryPoint,
+			String authority) {
+		this.missingAuthoritiesHandlerBuilder.addEntryPointFor(entryPoint, authority);
+		return this;
+	}
+
+	public ExceptionHandlingConfigurer<H> defaultAuthenticationEntryPointFor(
```

Comment:
This name is a bit confusing to me because it is configuring the access denied handler and does not clarify what the String type is. Perhaps something like `defaultDeniedHandlerForMissingAuthority`.

NOTE We had talked about naming it based on the `Consumer<B>` type, but `B` is being passed into another builder which is building an `AuthorizationDeniedHandler`.